### PR TITLE
add workspace path to exposed properties

### LINKED ISSUE #4: Add workspace path to the exposed properties.

### PR DESCRIPTION
**Pull request by Foxcapades**
Resolves #4

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -45,6 +45,9 @@ dependencies {
 java {
   sourceCompatibility = JavaVersion.VERSION_16
   targetCompatibility = JavaVersion.VERSION_16
+
+  withJavadocJar()
+  withSourcesJar()
```
Comment (Foxcapades) [inline build.gradle.kts:50]: Unrelated change, should have been here all along.

**Review by ryanrdoherty — APPROVED** [review]
